### ISSUE TITLE
Glyphicons Errors

### ISSUE THREAD

**Issue by Robin481** [puzzle/skills#332]
When loading '/people/:id/skills' there is a loading error caused by the 'glyphicons-halflings' (the stars).
The first one happens when loading the page, the second one when hovering over the stars.
This error appears only in the browser's console. 

Definition of Done: 
- Loading error doesn't happen anymore when accessing a Person's Skills and on-hover
- Timebox: 2 hours

Edit:
I think I see a 401 Error in the console and the message `no JWT Token provided` which would lead us to Keycloak Shenanigangs.

**Comment by glowseeker** [comment]
So far I've spent about ~4 hours out of the 6 timeboxed for this issue and this is what I came up with:

- For both errors printed to the console, the cause seems to be the fact that the resources necessary (Glyphicons) can't be pulled, even though they seem to work without any flaws.


- The first error is linked to bootstrap-slider.js, explicitly line 385 where the issue is also GET error:
```
if (this.options.rtl === 'auto') 
{ this.options.rtl = window.getComputedStyle(this.element).direction === 'rtl'; }
```
 this seemed a little off to me, since this looks like it merely controls orientation with "right-to-left" and "left-to-right" writing systems and has no obvious correlation to the icons.


- The second error simply points to "glyphicon-halflings-005580" which didn't give me too big of a starting point


Again, I didn't spend the full 6 hours simply because I didn't get anywhere with a wide range of google searches. All in all, I'm not sure whether this is a very unintuitive bug or I just don't know enough about the subject to make any progress.
